index.html

```diff
@@ -0,0 +1,85 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link 
+        href="https://fonts.googleapis.com/css2?family=Inter:wght@500&family=Poppins:wght@600&display=swap"
+        rel="stylesheet">
+</head>
+<body>
+    <header >
+        <nav class="nav-bar">
+            <img
+              class="logo"
+              src="images/logo.png"
+              alt="This is the logo of my portfolio webpage"
+            />
+            <ul class="links-holder">
+              <li class="link"><a href="index.html">Portoflio</a></li>
+              <li class="link"><a href="#">About</a></li>
+              <li class="link"><a href="#">Contact</a></li>
+            </ul>
+            <div class="hamburger">
+              <span class="line"></span>
+              <span class="line"></span>
+              <span class="line"> </span>
+            </div>
+          </nav>
+        <main>
+            <!--Headline section-->
+            <section class="headline-section">
+                <div>
+                  <h1 tabindex="0">I'm Racha Glad to see you!</h1>
+                  <p tabindex="0" class="about-me">
+                    I’m a software developer! I can help you build a product , feature
+                    or website Look through some of my work and experience! If you like
+                    what you see and have a project you need coded, don’t hestiate to
+                    contact me.
+                  </p>
+                  <a class="connect" href="#">LET’S CONNECT</a>
+                  <ul class="social-icons">
+                    <li>
+                      <a href="#"
+                        ><img class="icon" src="images/twitter.svg" alt="Twitter" />
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#">
+                        <img class="icon" src="images/linked-in.svg" alt="Linkedin" />
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#">
+                        <img class="icon" src="images/just-m.svg" alt="just-m" />
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#">
+                        <img class="icon" src="images/github.svg" alt="github" />
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#"
+                        ><img class="icon" src="images/finger.svg" alt="finger" />
+                      </a>
+                    </li>
+                  </ul>
+                </div>
+            </section>
```

Comment:
Nice job on creating your headline section😉 but try to stick to the given design😉 as this is a requirement for this milestone😎 kindly see the image comparison below😉

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="282" alt="your headline" src="https://user-images.githubusercontent.com/64643995/227227008-6ce52dc7-3596-49e4-9226-ef25e9b6a320.png" width="300">|![headline-mobile](https://user-images.githubusercontent.com/64643995/227227122-dca28ce1-718a-4918-abd4-00ed640ee0a7.png)|

Kindly take note of the `font-properties`  and the `spacings` and the `background image` and make the necessary change😉